lib/dictBuilder/cover.rs

```diff
@@ -1210,7 +1210,7 @@ fn COVER_tryParameters(data: Box<COVER_tryParameters_data_t>) {
 ///   of [`core::slice::from_raw_parts`].
 /// - `sum(samplesSizes)` is 0 or `samplesBuffer` and `sum(samplesSizes)` satisfy the requirements
 ///   of [`core::slice::from_raw_parts`].
-/// - `parameters` satisfies the requirements of [`pointer::as_mut`]
+/// - `parameters` satisfies the requirements of `pointer::as_mut`
```

Comment:
in zlib we provide the link ourselves, e.g. 

```md
[`pointer::as_mut`]: link to that method
```

each markdown block using that trick does need to redefine the link unfortunately, but I think it's a better experience for users.